src/app/_components/DesktopNavigation.tsx

```diff
@@ -0,0 +1,181 @@
+'use client'
+
+import Link from 'next/link'
+import { usePathname } from 'next/navigation'
+
+import { ArrowUpRight } from '@phosphor-icons/react'
+import clsx from 'clsx'
+import { Route } from 'next'
+
+import { Icon } from '@/components/Icon'
+import { NavigationPopover } from '@/components/NavigationPopover'
+
+import { PATHS } from '@/constants/paths'
+
+const getInvolvedItems = [
+  {
+    label: PATHS.EVENTS.label,
+    href: PATHS.EVENTS.path,
+    description: 'Overview of upcoming and past events',
+  },
+  {
+    label: PATHS.GRANTS.label,
+    href: PATHS.GRANTS.path,
+    description:
+      'Information on funding opportunities supporting projects that contribute to the growth of the Filecoin Network',
+  },
+  {
+    label: 'Become a Storage Provider',
+    href: 'https://destor.com/destor-network/overview',
+    description: 'Dive right in and become an essential part of the ecosystem',
+  },
+  {
+    label: 'Contribute to the Filecoin Project',
+    href: 'https://docs.filecoin.io/basics/project-and-community/ways-to-contribute',
+    description:
+      'Shape the future of Filecoin by contributing to its code, research, or docs.',
+  },
+]
+
+const communityItems = [
+  {
+    label: PATHS.ECOSYSTEM.label,
```

Comment:
Should be `Ecosystem Explorer`

Reply:
Should we update it in the `PATHS` object?